services/workflows-service/scripts/clean.ts

```diff
@@ -18,7 +18,7 @@ async function clean() {
   const tables = await getTables(prisma);
   const types = await getTypes(prisma);
   await dropTables(prisma, tables);
-  await dropTypes(prisma, types);
+  await dropTypes(prisma, types.sort());
```

Comment:
The sort here is a workaround for an error that happens on the db clean step.

Reply:
I suggest finding a more reliable way of ensuring the order in which the types are getting removed (dependency wise). I still feel like `clean.ts` can be boiled down to some drop db operation or `TRUNCATE`. Prisma might have an API for cleaning the db.